portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -37,60 +39,92 @@
   private static final String REFLECTION_PARAMETER = "reflection";
   private static final String COMMENT_PARAMETER = "Comment";
   private static final String TIMESTAMP_PARAMETER = "timestamp";
+  private static final String MAXCOMMENT_PARAMETER = "max-comment";
   private static final String APPLICATION_JSON_PARAMETER = "application/json;";
   private static final String COMMENT_HTML_PARAMETER = "/comments.html";
-
+  private static final String COMMENT_FORM = "comment-form";
+  private static final String MAX_FORM = "max-form";
 
   private String userNameInput;
   private String reflectionInput;
-  private long idGiven;
-  private long timestampOfComment;  
+  private String timestampOfComment;
+  private long idGiven; 
+  private int maxComment = 5; 
 
   public List<Comment> comments;
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    this.userNameInput = request.getParameter(this.USERNAME_PARAMETER);
-    this.reflectionInput = request.getParameter(this.REFLECTION_PARAMETER);
-    this.timestampOfComment = System.currentTimeMillis();
+    userNameInput = request.getParameter(this.USERNAME_PARAMETER);
+    reflectionInput = request.getParameter(this.REFLECTION_PARAMETER);
+    timestampOfComment = getCurrentTime();
```

Comment:
This can be `timestampOfComment = LocalDateTime.now()`

Reply:
I wanted to be formated to follow a specific pattern tough